app/views/namespaces/show.html.slim

```diff
@@ -45,14 +45,63 @@
 
 .panel.panel-default
   .panel-heading
-    h5
-      'Namespace:
-      strong
-        = @namespace.clean_name
-    - unless @namespace.global?
-      h6.label.label-info#team-label
-        | <span>Belongs to: </span>
-        = link_to "#{@namespace.team.name}", @namespace.team
+    .row
+      .col-xs-6
+        h5
+          'Namespace:
+          strong
+            = @namespace.clean_name
+        - unless @namespace.global?
+          h6.label.label-info#team-label
+            | <span>Belongs to: </span>
+            = link_to "#{@namespace.team.name}", @namespace.team
+      .col-xs-6.text-right
+        - if can_push?(@namespace)
+          .circle-label.permissions-label.circle-label-sm[data-placement="left"
+            data-toggle="popover"
+            data-container=".panel-heading"
+            data-content="You can push images"
+            data-original-title="What can I do?"
+            tabindex="0" data-html="true"]
+              i.fa.fa-arrow-up
+              ' Push
+        - if can_pull?(@namespace)
+          .circle-label.permissions-label.circle-label-sm[data-placement="left"
+            data-toggle="popover"
+            data-container=".panel-heading"
+            data-content="You can pull images"
+            data-original-title="What can I do?"
+            tabindex="0" data-html="true"]
+            i.fa.fa-arrow-down
+            ' Pull
+        - if owner?(@namespace)
+          .circle-label.permissions-label.circle-label-sm[data-placement="left"
+            data-toggle="popover"
+            data-container=".panel-heading"
+            data-content="You are the owner of this namespace"
```

Comment:
nitpicking, but this should be "You are an owner of this namespace" since there can be more than just one owner.